playbooks/common/openshift-cluster/upgrades/v3_3/master_config_upgrade.yml

```diff
@@ -48,3 +48,7 @@
     dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
     yaml_key: 'controllerConfig.servicesServingCert.signer.keyFile'
     yaml_value: service-signer.key
+- modify_yaml:
+    dest: "{{ openshift.common.config_base}}/master/master-config.yaml"
+    yaml_key: 'admissionConfig.pluginConfig'
+    yaml_value: "{{ openshift.master.kube_admission_plugin_config | to_padded_yaml(level=2) }}"
```

Comment:
The docs seem to indicate we should have been doing this starting with 1.3 (https://docs.openshift.org/latest/architecture/additional_concepts/admission_controllers.html#admission-controllers-general-admission-rules) so I thought it would be good to do on minor upgrades too. Though I guess that does increase the chances of something breaking…